app/js/controllers/main.js

```diff
@@ -23,6 +25,20 @@ export default class MainController extends BaseController {
       'reminders': remindersController,
     };
 
+    const speechController = new SpeechController();
+    speechController.on(
```

Comment:
At the minute, I'm not certain of the best way to send these to the correct UI (currently active controller) to display something - is there something simple we can do?